skbio/alignment/_tabular_msa.py

```diff
@@ -1647,6 +1647,7 @@ def gap_frequencies(self, axis='sequence', relative=False):
             gap_freqs.append(sum(freqs.values()))
 
         gap_freqs = np.asarray(gap_freqs, dtype=float if relative else int)
+        gap_freqs = gap_freqs[~np.isnan(gap_freqs)]
```

Comment:
If we agree, this is the "easiest" solution for the warnings about: true_divide,  greater, less, etc